CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-### 5.6.0 / 2018-09-09
+### 5.6.0 / 2018-09-26
 * Add support for Beaker 4
 
+### 5.5.4 / 2018-09-24
```

Comment:
This is not being released independent of the Beaker 4 above, so it should have the same version number